pkg/util/detector/detector.go

```diff
@@ -1038,3 +1121,26 @@ func (d *ResourceDetector) CleanupLabels(objRef workv1alpha1.ObjectReference, la
 	klog.V(2).Infof("Updated resource(kind=%s, %s/%s) on cluster: %s", newWorkload.GetKind(), newWorkload.GetNamespace(), newWorkload.GetName(), newWorkload.GetClusterName())
 	return nil
 }
+
+// requestResource is a collection of compute resource.
+type requestResource struct {
+	MilliCPU int64
+	Memory   int64
+}
+
+func (r *requestResource) addResource(rl corev1.ResourceList) {
```

Comment:
Why not use the struct and funtions at  https://github.com/karmada-io/karmada/blob/26b9fcb272b583f53039d1cf6c4c5a19c2212a40/pkg/util/resource.go#L6

Reply:
I do not notice the struct already existed. I will modify it.